### ISSUE TITLE
Dial-in phone number options when using default meet.jit.si server

### ISSUE THREAD

**Issue by adamzea** [timetheoretical/jitsi-meet-outlook#28]
It looks like there are some API endpoints for pulling plain-old-telephone numbers that can allow users to dial-in to the conference call.  Maybe those could be used to include "join by telephone" numbers in the meeting notes below the join-meeting URL.

https://jitsi.org/api/

```
Adding a one-tap number
The next step does a couple of API calls but still no authentication or authorization necessary. Let’s say that in every event you want to add a one tap dial-in number that looks like this:
+1 209-844-4600,,1366520583#
In order to do that you need to run this API call first:
https://api.jitsi.net/phoneNumberList?conference=SuccessivePortsCompeteMeanwhile@conference.meet.jit.si
This will return all the phone numbers that can be used to join this meeting and the first one will always be the one we detected as closest to the endpoint that did the call. We run this on in the browser so that we can get a number that matches the location of the user.
The second call you might want to do is one that will give you the pin code for the meeting:
https://api.jitsi.net/conferenceMapper?conference=some-id-you-like@conference.mycompany.meet.jit.si
With that you get the pin.
And that’s all! Let us know if you have any questions or if you want to jump on a call and talk about this!
```

I don't even know if it's possible to recognize if a self-hosted Jitsi Meet server has the dial-in capability installed, which is why this suggestion is only for when using the default meet.jit.si server.

**Comment by timetheoretical** [comment]
Thanks for pointing this out. This would be nice to include.

> I don't even know if it's possible to recognize if a self-hosted Jitsi Meet server has the dial-in capability installed, which is why this suggestion is only for when using the default meet.jit.si server.

At least the way I've structured the add-on for now, there is no actual communication with the Jitsi server. The plugin only generates a url.

As mentioned in #30, I do not have much time to spend on this in the nearer future. But I will add this as a longer-term goal.

**Comment by timetheoretical** [comment]
This was implemented in version 0.7.0, see #58.
